manipulation_action_server_executor:
  ros__parameters:
    ik_frame: gripper_grasping_frame
    group: arm_torso
    eef: gripper
    arm_group: arm_torso
    gripper_group: gripper
    open_pose: open
    close_pose: close
    gripper_joints: ['gripper_left_finger_joint', 'gripper_right_finger_joint']
    gripper_closed: [0.0, 0.0]
    gripper_open: [0.04, 0.04]
    gripper_tolerances: [0.01, 0.01]

    pick_from_pc:
      retreat_axis: "y"
      retreat_min: 0.05 # middle of the gripper
      retreat_max: 0.1  # end of the gripper
      retreat_resolution: 0.01 # distance to st
      config_file: "/home/juan/workspaces/robocup/src/manipulation_server/manipulation_action_server/params/ros_vino_params.cfg"
     

    
